### PR TITLE
url for articles; remove arxiv url

### PR DESCRIPTION
**Pull request by kohpangwei**

### CODE REVIEW

File: data/venues.rb [hunk 1]
```diff
@@ -34,8 +34,8 @@ def dukeStat(year); techreport(Name.new('Department of Statistics, Duke Universi
 def berkeleyStat(year); techreport(Name.new('Department of Statistics, University of California at Berkeley', 'UC Berkeley'), year) end
 def berkeleyEECS(year); techreport(Name.new('EECS Department, University of California at Berkeley', 'UC Berkeley'), year) end
 def mit(year); techreport('Massachusetts Institute of Technology', year) end
-def arxiv(year, id=nil); [article('arXiv' + (id != nil ? ' preprint arXiv:'+id : ''), year, nil), id && url('https://arxiv.org/pdf/' + id + '.pdf')].compact end
-# def arxiv(year, id=nil); article('arXiv' + (id != nil ? ' preprint arXiv:'+id : ''), year, nil) end
+# def arxiv(year, id=nil); [article('arXiv' + (id != nil ? ' preprint arXiv:'+id : ''), year, nil), id && url('https://arxiv.org/pdf/' + id + '.pdf')].compact end
+def arxiv(year, id=nil); article('arXiv' + (id != nil ? ' preprint arXiv:'+id : ''), year, nil) end
```
Comment (percyliang) [inline data/venues.rb:38]: Could you leave the URL in for arxiv papers?  This is useful for the HTML pages, where arxiv papers get URLs automatically.

Comment (kohpangwei) [reply data/venues.rb:38]: That will add URLs to all bibliography entries that cite arxiv papers -- is that ok? (We typically don't provide URLs for other papers) Or is there some flag we could set to show URLs only on particular entries? `howpublished` is another reasonable field to put the URL in, and it makes it into all.bib, but the style file seems to strip that out...

Comment (percyliang) [reply data/venues.rb:38]: Yeah, we don't want the URL everywhere.  You could make another custom field `visibleUrl` (analogous to `extendedVersion`) and then only include the `url` in `toBibtex` if `visibleUrl` is set.

Comment (kohpangwei) [reply data/venues.rb:38]: Ok, added `visibleurl` (kept lower case for consistency with the other url fields). PTAL.

Actually... the other url fields are strings and this is a boolean, so maybe `isVisibleUrl`? Though then that's not consistent with `extendedVersion`.

Comment (percyliang) [reply data/venues.rb:38]: How about `isUrlVisible`?

**Review by percyliang — APPROVED** [review]
LGTM - thanks!

**Pull request merged**